pandas/tests/groupby/test_numeric_only.py

```diff
@@ -368,18 +368,11 @@ def test_deprecate_numeric_only_series(dtype, groupby_func, request):
             msg = "cannot be performed against 'object' dtypes"
         else:
             msg = "is not supported for object dtype"
-        warn = FutureWarning if groupby_func == "fillna" else None
-        warn_msg = "DataFrameGroupBy.fillna is deprecated"
-        with tm.assert_produces_warning(warn, match=warn_msg):
-            with pytest.raises(TypeError, match=msg):
-                method(*args)
+        with pytest.raises(TypeError, match=msg):
+            method(*args)
     elif dtype is object:
-        warn = FutureWarning if groupby_func == "fillna" else None
-        warn_msg = "SeriesGroupBy.fillna is deprecated"
-        with tm.assert_produces_warning(warn, match=warn_msg):
-            result = method(*args)
-        with tm.assert_produces_warning(warn, match=warn_msg):
-            expected = expected_method(*args)
+        result = method(*args)
+        expected = expected_method(*args)
```

Comment:
Those are not related to quantile with bool, right? I'm a bit confused here.

Reply:
Yea - this was left over from #57109; the `fillna` method no longer exists on groupby.